clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -36,6 +36,7 @@
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Modifier;
+import java.net.Socket;
```

Comment:
Maybe this belongs as a utility method in Selector, since it's specific to the network subsystem, and this ID is meant to be used in Selector.